source/views/menus/lib/trim-names.js

```diff
@@ -5,7 +5,17 @@ export function trimStationName(stationName: string) {
   return stationName.replace(/<strong>@(.*)<\/strong>/, '$1')
 }
 
+function removeParenTags(str: string) {
+  const parensRegex = / \([^)]*?\)$/
+  while (str !== str.replace(parensRegex, '')) {
```

Comment:
I just had a thought:

Instead of running the regex replacement and doing a string comparison, why not just use the `RegExp#test` function?

I tested this in [Runkit](https://runkit.com/hawkrives/58b1ca46e302c300142e1caa) and it works just like the comparison version.

```diff
-while (str !== str.replace(parensRegex, '')) {
+while (parensRegex.test(str)) {
```